### PR TITLE
List & detail payments endpoint

### PR DESCRIPTION
**Pull request by AVyha**
10 task

### CODE REVIEW

File: payment/serializers.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    class Meta:
+        model = Payment
+        fields = (
+            "id",
+            "status",
+            "type",
+            "borrowing",
+            "session_url",
+            "session_id",
+            "money",
```
Comment (mltdll) [inline payment/serializers.py:16]: I think `money` should be a read-only field that is calculated automatically.

File: payment/views.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+from rest_framework import viewsets, generics, mixins
+from rest_framework.permissions import IsAuthenticated
+
+from payment.models import Payment
+from payment.serializers import PaymentSerializer
+
+
+class PaymentViewSet(
```
Comment (mltdll) [inline payment/views.py:8]: I guess it makes sense to allow users access to detail views of their payments only.

**Review by mltdll — CHANGES_REQUESTED** [review]

File: payment/serializers.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        fields = (
+            "id",
+            "status_payment",
+            "type_payment",
+            "borrowing",
+            "session_url",
+            "session_id",
+            "money",
+        )
+        read_only_fields = ("id", "money",)
```
Comment (mltdll) [inline payment/serializers.py:18]: But the `money` field should also be calculated somewhere. Or are you going to implement this in some of the following tasks?

**Review by mltdll — CHANGES_REQUESTED** [review]

File: payment/serializers.py (hunk 3 above)
Comment (AVyha) [reply payment/serializers.py:18]: in next task)
I understand that in this task i need implement only model/serializer/views

**Review by mltdll — APPROVED** [review]

**Review by Ilyakson — APPROVED** [review]

**Pull request merged**